layouts/partials/sharing-links.html

```diff
@@ -1,8 +1,7 @@
 {{ with .Params.sharingLinks | default (.Site.Params.article.sharingLinks | default false) }}
-  {{ $links := site.Data.sharing }}
   <section class="flex flex-row flex-wrap justify-center pt-4 text-xl">
-    {{ range . }}
-      {{ with index $links . }}
+    {{ range $.Site.Data.sharing.links }}
```

Comment:
I don't think if there's a huge difference between this and `site.Data.sharing`, but only `$.Site.Data.sharing.links` worked for me. This also seems to be the [recommended way](https://gohugo.io/templates/data-templates/) to access data.